src/linker/pipeline.py

```diff
@@ -121,17 +121,24 @@ def write_implementation_rules(
         validation_file = str(
             results_dir / "input_validations" / implementation.validation_filename
         )
+        resources = (
```

Comment:
nit: we may want to specify this as "slurm_resources" (as opposed to spark resources or in the future who knows what else)